Net/src/WebSocketImpl.cpp

```diff
@@ -297,7 +297,7 @@ int WebSocketImpl::receiveBytes(void* buffer, int length, int)
 
 		skipHeader(_receiveState.headerLength);
 
-		if (receivePayload(reinterpret_cast<char*>(buffer), payloadLength, _receiveState.mask, _receiveState.useMask) != payloadLength)
+		if (receivePayload(reinterpret_cast<char*>(buffer), payloadLength, _receiveState.mask, _receiveState.useMask, 0) != payloadLength)
```

Comment:
For consistency I would pass in `_receiveState.maskOffset` here as well instead of 0.